src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -1158,4 +1162,24 @@ function (Document $function) use ($deleteByFunction) {
             );
         }
     }
+
+    /**
+     * @param Database $dbForConsole
+     * @param callable $getProjectDB
+     * @param Document $team
+     * @param Document $project
+     * @throws Authorization
+     * @throws Conflict
+     * @throws DatabaseException
+     * @throws Exception
+     * @throws Restricted
+     * @throws Structure
+     */
+    public function deleteTeam(Database $dbForConsole, callable $getProjectDB, Document $team, Document $project): void
+    {
+        $this->deleteMemberships($getProjectDB, $team, $project);
+        if ($project->getId() === 'console') {
+            $this->deleteProjectsByTeam($dbForConsole, $getProjectDB, $team);
```

Comment:
Deleting projects synchronously during the API call can take some time, right? Can we avoid that?